utils/test-utils/README.md

```diff
@@ -0,0 +1,254 @@
+By default, not passing parameters and using the default ones, the folding feature will be centered, have 0px width and the position of the device will be HALF_OPENED (book mode).
+
+```
+fun <A : ComponentActivity> TestRule.createHorizontalFoldingFeature(
+    activityRule: ActivityScenarioRule<A>,
+    center: Int = -1,
+    size: Int = 0,
+    state: FoldingFeature.State = FoldingFeature.State.HALF_OPENED)
+```
+Creates an horizontal folding feature with the defined size, center position and state. By default, not passing parameters and using the default ones, the folding feature will be centered, have 0px width and the position of the device will be HALF_OPENED (table-top/desktop mode).
+
+```
+fun <A : ComponentActivity> TestRule.createSurfaceDuo1FoldingFeature(
+    activityRule: ActivityScenarioRule<A>,
+    state: FoldingFeature.State = FoldingFeature.State.FLAT)
+```
+Simulates Surface Duo 1 folding feature (84px FoldingFeature width). By default, the position of the of the device will be FLAT (open in 180 degrees).
+
+```
+fun <A : ComponentActivity> TestRule.createSurfaceDuo2FoldingFeature(
+    activityRule: ActivityScenarioRule<A>,
+    state: FoldingFeature.State = FoldingFeature.State.FLAT)
+```
+Simulates Surface Duo 2 folding feature (66px FoldingFeature width). By default, the position of the of the device will be FLAT (open in 180 degrees).
+
+```
+fun <A : ComponentActivity> TestRule.createFoldWithVerticalFoldingFeature(
+    activityRule: ActivityScenarioRule<A>,
+    state: FoldingFeature.State = FoldingFeature.State.HALF_OPENED)
+```
+Simulates well known available foldable device's vertical folding feature (0px FoldingFeature width). By default, the position of the of the device will be HALF_OPENED (book mode).
+
+```
+fun <A : ComponentActivity> TestRule.createFoldWithHorizontalFoldingFeature(
+    activityRule: ActivityScenarioRule<A>,
+    state: FoldingFeature.State = FoldingFeature.State.HALF_OPENED)
+```
+Simulates well known available foldable device's horizontal folding feature (0px FoldingFeature width). By default, the position of the of the device will be HALF_OPENED (table-top/desktop mode).
+
+### ForceClick
+
+`class ForceClick : ViewAction`
+
+This is a [ViewAction](https://developer.android.com/reference/androidx/test/espresso/ViewAction) that clicks on a given [View](https://developer.android.com/reference/android/view/View) without checking its coordinates in the screen when the device has been rotated.
+
+### SurfaceDuoUtils
+
+Contains useful specs information and utilities for tests that target **Surface Duo** devices.
+
+`object SurfaceDuo1`
+
+Inside this object you can find useful information about **Surface Duo 1** specs and utilities to span and unspan apps on this device and emulator.
+
+`fun switchFromSingleToDualScreen()`
+
+Span an app across displays. The function will handle correctly the different rotation modes of the device.
+This function works on **Surface Duo 2**.
```

Comment:
I think here it should also be "Surface Duo **1**"

Reply:
wops!! good catch!

Reply:
Fixed